apps/homescreen/js/app.js

```diff
@@ -401,14 +424,19 @@
           if (child.order !== -1 && child.order < container.order) {
             continue;
           }
-          this.icons.insertBefore(container, child,
-                                  this.iconAdded.bind(this, container));
+          this.icons.insertBefore(container, child);
```

Comment:
This would be a gaia-container change - I think having the callback is a generally useful thing, we may well use it later on in other optimisations (or just to trigger events). I'm against removing it for now, but if it doesn't prove to be useful at any point in the near future, let's consider it then.